### ISSUE TITLE
112RM-SL adult atlas

### ISSUE THREAD

**Issue by ravenep** [PRIME-RE/RheMAP#6]
Hello! 

I saw you mentioned the 112RM-SL adult atlas in the summary table but I do not see it as an option in the transformation list (maybe I am missing it?) 

Do you have plans to include this? We have some nice legacy data and code at the lab that references this atlas and I am hoping to get it into NMT space to interface with the CIVET surface pipeline. 

Thanks for any input & hope all is well. 

Cheers, 
Erika

**Comment by pcklink** [comment]
Isn't the 112RM-SL atlas in D99 space? If not, it would be straightforward to run it if you can point us to an official download. If it is, you can use the warps between D99 and NMT to warp.

**Comment by ravenep** [comment]
Ok! Thanks for the comment. I will look into this more and reach out if we have any issues. Good to know there can be an additional warp if needed. Thanks so much, 
Erika

**Comment by ravenep** [comment]
So, this is very old now, but I do have some insights. 

I had been trying to figure out what the mysterious 112RM-F99 atlas was in, or any details about it to ultimately translate results to the NMT template.

- The world of atlases is confusing. But, I know from this paper (Table 1 / Macaque https://pubmed.ncbi.nlm.nih.gov/17964241/) that F99 is a single subject surface based atlas. This was made by Van Essen and used in Caret software in the early 2000s.

- From ~ 2006 another single subject atlas - the D99 made by Saleem / Logothetis, came to the scene. So, are the F99 and D99 in the same space? No.

- Then there is the 112RM. This comes from Wash U by Donald McLeren / Sterling Johnson (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2659879/) The 112 is a population based atlas (eighty-two male and thirty female rhesus macaques) which is aligned with D99 (and thus takes advantage of histology from D99 ). They acknowledged however that the F99 had surfaces, so they included transformations to that template.

There are probably links out there, but I have not found anything live that includes the transforms. You said it might be possible to add another member to RHE map - if this is still an option, would you consider F99? There is a download available with XTRACT software from FSL https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/XTRACT

We are past Easter, but similar to an egg hunt!

**Comment by ravenep** [comment]
tldr; I realize this is still very confusing to follow , but I am hoping to achieve F99 -> NMT

**Comment by pcklink** [comment]
Sure. I'll have a look when I have some time. I think I may have a F99 somewhere, but will also check what is hidden in FSL. May take a while. You can always download the RheMap repo and run it yourself if it takes too long.

**Comment by ravenep** [comment]
thanks @pcklink !

**Comment by pcklink** [comment]
I had a bit of time last evening and ran it overnight. The results were just uploaded to [https://gin.g-node.org/ChrisKlink/RheMAP](https://gin.g-node.org/ChrisKlink/RheMAP). You should be able to download them there.

**Comment by ravenep** [comment]
that's great! thank you for including it!
